src/web/aws/cloudwatch/StepReview.jsx

```diff
@@ -59,7 +58,7 @@ const StepReview = ({ onSubmit, onEditClick, externalInputSubmit }) => {
     awsEndpointKinesis = { value: undefined },
   } = formData;
 
-  const globalInputEnabled = !!awsCloudWatchGlobalInput.value;
+  const globalInputEnabled = true;
```

Comment:
I'd remove this variable completely and replace it's usage with the appropriate values.